core/src/test/java/org/mskcc/cbio/oncokb/util/AminoAcidConverterUtilsTest.java

```diff
@@ -0,0 +1,40 @@
+package org.mskcc.cbio.oncokb.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AminoAcidConverterUtilsTest {
```

Comment:
Also we need to include some tests that will not be converted from 3letter to 1letter. `Promoter` is a good test case.